google-cloud-contrib/google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/CloudStorageReadChannel.java

```diff
@@ -136,8 +141,13 @@ public int read(ByteBuffer dst) throws IOException {
   }
 
   private void sleepForAttempt(int attempt) {
+    long delay = 500;
+    // exponential backoff
```

Comment:
It has, yes, though I haven't been able to run the actual GenomicsDBImport. I was relying on someone else to run through my [test instructions](https://github.com/broadinstitute/gatk/issues/2685#issuecomment-302798685) before sending the PR for review, but the timeline's been compressed.

For extra peace of mind I've doubled the initial delay to one second. These errors represent abnormal behavior so we can make a case that a long delay is reasonable.